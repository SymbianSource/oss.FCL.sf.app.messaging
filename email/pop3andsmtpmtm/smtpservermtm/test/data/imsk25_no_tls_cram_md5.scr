220 dummy.test.com ESMTP 
250-dummy.test.com is pleased to meet you
250-HELP
250-PIPELINING
250-ETRN
250-DSN
250-ATRN
250-STARTTLS
250-SIZE
250-AUTH=LOGIN
250-AUTH LOGIN PLAIN CRAM-MD5 DIGEST-MD5
250 EHLO
334 PDQ2MDg5LjEwMjU4NjA0NjZAdHJtYWkwMi5pdC5jZWxsdWxhcmRhdGEuY29tPg==
235 rahman relaying authenticated
250 SMTP state reset
250 test@dummy.test.com sender accepted
250 testrcpt@dummy.rcpt.test.com will relay mail for an authenticated user
354 Enter mail, end with "." on a line by itself
250 114136 message accepted for delivery
221 dummy.test.com CommuniGate Pro SMTP closing connection
