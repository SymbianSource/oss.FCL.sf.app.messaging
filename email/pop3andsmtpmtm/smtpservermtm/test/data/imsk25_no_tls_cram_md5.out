EHLO []
AUTH CRAM-MD5
c210cC1iYWQgYWM0ODIyZWIyYTkyZjRiOGFhMWE0YmE3ZGU1ZTRkNWY=
RSET
MAIL FROM:<smtp@lon-msgtest06.intra>
RCPT TO:<recyclebin@lon-msgtest06.intra>
DATA
From: "SMTP AUTH" <smtp@lon-msgtest06.intra>
Reply-to: "SMTP AUTH" <smtp@lon-msgtest06.intra>
To: recyclebin@lon-msgtest06.intra
Subject: Test message No.1 - 1000 character line in message
Date: date-replacement-string-
Message-ID: <message-id-replacement-string-1>
X-Mailer: EPOC Email Version 2.10
MIME-Version: 1.0
Content-Language: i-default
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: quoted-printable

The line in the following 3 paragraphs each contain 1000 characters: =

the maximum limit of linelength for SMTP mail. Each should arrive as
one wrapped paragraph in the received message. The dot-stuffed
paragraph is the absolute maximum line length permissable in SMTP
protocol - although in practice IMCV wraps the data into shorter
lines before it is actually sent.

1234567890123456789012345678901234567890123456789012345678901234567890123456=
7890123456789012345678901234567890123456789012345678901234567890123456789012=
3456789012345678901234567890123456789012345678901234567890123456789012345678=
9012345678901234567890123456789012345678901234567890123456789012345678901234=
5678901234567890123456789012345678901234567890123456789012345678901234567890=
1234567890123456789012345678901234567890123456789012345678901234567890123456=
7890123456789012345678901234567890123456789012345678901234567890123456789012=
3456789012345678901234567890123456789012345678901234567890123456789012345678=
9012345678901234567890123456789012345678901234567890123456789012345678901234=
5678901234567890123456789012345678901234567890123456789012345678901234567890=
1234567890123456789012345678901234567890123456789012345678901234567890123456=
7890123456789012345678901234567890123456789012345678901234567890123456789012=
3456789012345678901234567890123456789012345678901234567890123456789012345678=
901234567890

The following paragraph will be dot stuffed into 1001 characters:

..123456789012345678901234567890123456789012345678901234567890123456789012345=
6789012345678901234567890123456789012345678901234567890123456789012345678901=
2345678901234567890123456789012345678901234567890123456789012345678901234567=
8901234567890123456789012345678901234567890123456789012345678901234567890123=
4567890123456789012345678901234567890123456789012345678901234567890123456789=
0123456789012345678901234567890123456789012345678901234567890123456789012345=
6789012345678901234567890123456789012345678901234567890123456789012345678901=
2345678901234567890123456789012345678901234567890123456789012345678901234567=
8901234567890123456789012345678901234567890123456789012345678901234567890123=
4567890123456789012345678901234567890123456789012345678901234567890123456789=
0123456789012345678901234567890123456789012345678901234567890123456789012345=
6789012345678901234567890123456789012345678901234567890123456789012345678901=
2345678901234567890123456789012345678901234567890123456789012345678901234567=
890123456789

Start67890123456789012345678901234567890123456789012345678901234567890123456=
7890123456789012345678901234567890123456789012345678901234567890123456789012=
3456789012345678901234567890123456789012345678901234567890123456789012345678=
9012345678901234567890123456789012345678901234567890123456789012345678901234=
5678901234567890123456789012345678901234567890123456789012345678901234567890=
1234567890123456789012345678901234567890123456789012345678901234567890123456=
7890123456789012345678901234567890123456789012345678901234567890123456789012=
3456789012345678901234567890123456789012345678901234567890123456789012345678=
9012345678901234567890123456789012345678901234567890123456789012345678901234=
5678901234567890123456789012345678901234567890123456789012345678901234567890=
1234567890123456789012345678901234567890123456789012345678901234567890123456=
7890123456789012345678901234567890123456789012345678901234567890123456789012=
3456789012345678901234567890123456789012345678901234567890123456789012345678=
901234567End

End of message.

.
QUIT
